src/SoundList.js

```diff
@@ -9,64 +9,94 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+const SoundList = ({
```

Comment:
SoundList should not know anything about filtering. It should be a very simple component that just shows sounds. We handle our "business logic" complexity in Sound.js. Its tempting to pass `isSimilarSoundList` to `SoundList` but it's not architecturally sound.

Readings:
* https://en.wikipedia.org/wiki/Don%27t_repeat_yourself
* https://en.wikipedia.org/wiki/Single-responsibility_principle

An easy way to determine what logic should go where is the following: you should see our storybook components as a component library that can be consumed by anyone. If someone sees a soundlist component that they can consume as a library (say `react-soundlist`), then they should be able to use it. On the other hand, `Sound.js` isn't a component that is very generalizable, and this is what we intended. It contains a lot of functionality that is specific to Tide. You can see `Sound.js` as composing these reusable/generalized components like SoundList and passing them functionality that is application specific.

Reply:
That makes sense. The most recent commit at the time of this writing does the filtering work inside of `Sound.js`, and not `SoundList.js`.

Reply:
That makes sense. The most recent commit at the time of this writing does the filtering work inside of `Sound.js`, and not `SoundList.js`. Thanks for the feedback!